README.md

```diff
@@ -224,6 +224,8 @@ arguments.
         callbacks (for databases loaded with `locus_custom_fetcher``)
   - Caching can be disabled by specifying the `no_cache` option when
     running `:start_loader`
+  - The cache database location can be customised by providing
+    `{database_cache_file, FilePath}` option for `locus_loader`.
```

Comment:
```suggestion
    `{database_cache_file, FilePath}` option for `locus_loader`
    (`FilePath` must have a ".mmdb.gz" extension)
```

Reply:
Reason:

https://github.com/g-andrade/locus/blob/3037d490254634972bb26c234b450110173c3abd/src/locus_loader.erl#L939-L942

Reply:
:thinking:  ideally this constraint would be validated within `validate_loader_opts/2` but I'd consider that just a nice to have.

Reply:
Added validation, since there is already handy `filename_extension_parts/1` function.